### DISCUSSION TITLE
Keypress Handler

### DISCUSSION THREAD

**Discussion by Dooffy** [twilio-labs/paste#973] [Q&A] [answered]
Hey, is it possible to handle on key press with paste input?

**Comment by richbachman** [comment]
Hi @Dooffy 

Would you mind providing more context about what you're trying to accomplish? Either a screenshot or a codesandbox using Paste would be super helpful.

Thanks!

**Comment by Dooffy** [comment, marked as answer]
Solution:
```
<Box onKeyPress={onSubmitHandler}>
    <Input ... />
</Box>
```

The solution was shared with me to use `Box` key press handler. This one was hard to guess 😄 

The original issue was with a form submit by "Enter" like Google search:
<img width="702" alt="Screenshot 2020-11-25 at 15 18 15" src="https://user-images.githubusercontent.com/1443584/100246848-92ef6e80-2f31-11eb-9d5b-bfdf31f9bc91.png">

**Comment by TheSisb** [comment]
@Dooffy Glad you found an answer! As an aside, you can also wrap your inputs in a `<form onSubmit={}>` element and have a `<Button type="submit">` in the form, which will trigger the form's onSubmit when you hit Enter on the form.  This is native browser/HTML behavior.
